src/main/java/com/dev/cinema/model/Movie.java

```diff
@@ -0,0 +1,45 @@
+package com.dev.cinema.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Movie {
```

Comment:
```suggestion
public class Movie {
```
add here @ Table(name = "movie")
it's necessary  to name our table with lower case